### PR TITLE
Create feature toggles service

### PR DESCRIPTION
**Pull request by Luiz-FS**
**Feature/Bug description:** Create an initial scope for microservice feature management.

**Solution:** Create the initial layout of the feature management service.

**TODO/FIXME:** n/a

### CODE REVIEW

File: ecis [hunk 1]
```diff
@@ -212,6 +227,7 @@ case "$1" in
         set_landingpage_url local $LANDINGPAGE_LOCAL
         set_support_url local $SUPPORT_LOCAL
         set_frontend_url local $FRONTEND_LOCAL
+        set_feature_url local $FRONTEND_LOCAL
```
Comment (JuliePessoa) [inline ecis]: não seria `$FEATURE_LOCAL`?

File: ecis [hunk 2]
```diff
@@ -295,12 +311,14 @@ case "$1" in
                     APP_NAME="development-cis"
                     SUPPORT_DOMAIN="support-dot-$APP_NAME.appspot.com"
                     FRONTEND_DOMAIN="frontend-dot-$APP_NAME.appspot.com"
+                    FEATURE_DOMAIN="frontend-dot-$APP_NAME.appspot.com"
```
Comment (JuliePessoa) [inline ecis]: frontend?

File: ecis [hunk 3]
```diff
@@ -295,12 +311,14 @@ case "$1" in
...
                     break
                     ;;
                 "eciis-splab")
                     APP_NAME="eciis-splab"
                     SUPPORT_DOMAIN="support.plataformacis.org"
                     FRONTEND_DOMAIN="frontend.plataformacis.org"
+                    FEATURE_DOMAIN="frontend.plataformacis.org"
```
Comment (JuliePessoa) [inline ecis]: frontend?

File: feature-toggles/manage/manageTogglesService.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+(function() {
+    'use strict';
+
+    const app = angular.module('app');
+
+    app.service('ManageTogglesService', function($http) {
+        const service = this;
+        const uri = 'api/feature-toggles';
+
+        service.getFeatureToggles = function() {
```
Comment (JuliePessoa) [inline feature-toggles/manage/manageTogglesService.js]: getAll?

File: feature-toggles/manage/manageTogglesService.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+(function() {
+    'use strict';
+
+    const app = angular.module('app');
+
+    app.service('ManageTogglesService', function($http) {
+        const service = this;
+        const uri = 'api/feature-toggles';
```
Comment (ruanres) [inline feature-toggles/manage/manageTogglesService.js]: caps

File: feature-toggles/manage/manageToggles.js [hunk 6]
```diff
@@ -0,0 +1,9 @@
+(function() {
+    'use strict';
+
+    const app = angular.module('app');
+
+    app.controller('ManageTogglesController', function() {
+        const ManageTogglesCtrl = this;
```
Comment (ruanres) [inline feature-toggles/manage/manageToggles.js]: manageTogglesCtrl

**Pull request merged**